### DISCUSSION TITLE
Advanced Matrix Extensions support

### DISCUSSION THREAD

**Discussion by MineCake147E** [dotnet/runtime#87542] [Ideas]
Is there any plan or possibility of implementing AMX support in .NET?
I think it's important to have AMX support for aiming to be useful in research and application of Artificial Intelligence.

**Comment by huoyaoyuan** [comment]
It can be a blocker that most developers don't have a processor supports it.
AMX uses BF16 for floating point, which hasn't been added to BCL yet.
